project.clj

```diff
@@ -1,16 +1,17 @@
-(defproject refactor-nrepl "3.2.0"
+(defproject refactor-nrepl (or (not-empty (System/getenv "PROJECT_VERSION"))
```

Comment:
What's the point of the fallback version here? I get why the code is like this, it just look a bit confusing. Might be good to mention that the version will be set by `deploy-release` and just use `nil` as the placeholder in case PROJECT_VERSION is not set.

Reply:
Probably `nil` can cause problems wherever a string was expected, might have been the case from what I vaguely remember